src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorEditorFactory.cs

```diff
@@ -22,6 +22,8 @@ internal class RazorEditorFactory : EditorFactory
     {
         private const string EditorFactoryGuidString = "3dfdce9e-1799-4372-8aa6-d8e65182fdfc";
         private const string RazorLSPEditorFeatureFlag = "Razor.LSP.Editor";
+        private static readonly Guid LiveShareHostUIContextGuid = Guid.Parse("62de1aa5-70b0-4934-9324-680896466fe1");
```

Comment:
What are these guids? Where do they come from? Where do they go?

Reply:
These are hardcoded UIContext guids that LiveShare uses to indicate when various sessions are active. 

Alternatively i could have added a direct dependency to LiveShare to get these guids but utilizing guids in this fashion is pretty common practice for VS extensions to avoid unnecessary dependencies so I opted with this approach.